Source/Manager.swift

```diff
@@ -70,15 +71,19 @@ public class Manager: NSURLProtocol {
         }
     }
 
+    static public func removeResource(resource: ResourceProtocol) -> ResourceProtocol? {
+        return resources.remove(resource.resourceIdentifier)
```

Comment:
Will this ever return nil? If not, can the return type of the function be non-optional?